src/main/kotlin/pt/up/fe/ni/website/backend/controller/AuthController.kt

```diff
@@ -1,19 +1,22 @@
 package pt.up.fe.ni.website.backend.controller
 
 import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.security.core.context.SecurityContextHolder
 import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.PostMapping
 import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 import pt.up.fe.ni.website.backend.dto.auth.LoginDto
 import pt.up.fe.ni.website.backend.dto.auth.TokenDto
-import pt.up.fe.ni.website.backend.model.Account
+import pt.up.fe.ni.website.backend.model.Project
+import pt.up.fe.ni.website.backend.repository.ActivityRepository
 import pt.up.fe.ni.website.backend.service.AuthService
 
 @RestController
 @RequestMapping("/auth")
-class AuthController(val authService: AuthService) {
+class AuthController(val authService: AuthService, val repository: ActivityRepository<Project>) {
```

Comment:
```suggestion
class AuthController(val authService: AuthService, val projectRepository: ActivityRepository<Project>) {
```
more clear, since we're not in the project controller

Reply:
Apparently that variable is not being used, should we keep it?

Reply:
If it's not used, then no